runtime/shared_common/j9shr.tdf

```diff
@@ -2984,3 +2984,6 @@ TraceEvent=Trc_SHR_CM_storeCacheUniqueID_generateCacheUniqueID_after Overhead=1
 
 TraceEvent=Trc_SHR_CC_startup_getCacheUniqueID_before Overhead=1 Level=7 Template="CC::startup(): getCacheUniqueID() - (createTime: %llx, metadataBytes: %zx, classesBytes: %zx, lineNumTabBytes: %zx, varTabBytes: %zx) "
 TraceEvent=Trc_SHR_CC_startup_getCacheUniqueID_after Overhead=1 Level=7 Template="CC::startup(): getCacheUniqueID() - the cache unique ID is %s"
+
+TraceEvent=Trc_SHR_API_j9shr_classStoreTransaction_start_cacheFull_Event Overhead=1 Level=1 Template="API j9shr_classStoreTransaction_start : J9SHR_RUNTIMEFLAG_BLOCK_SPACE_FULL is set. The shared cache is full"
```

Comment:
Do these need to be level 1? They could be occurring quite often when the cache is full.

Reply:
I see the existing Trc_SHR_API_j9shr_createSharedClass_Full_Event is level 2 and cache could be full during the startup phase, so I made this trace point level 1. But I am fine with setting it level 3.

Reply:
Ideally there should be a single tracepoint when the cache becomes full, not a tracepoint every time a class is loaded when the cache is full.

Reply:
Changed to level 3.